core-api/src/main/java/com/optimizely/ab/config/parser/GsonHelpers.java

```diff
@@ -106,9 +106,13 @@ static Condition parseAudienceConditions(JsonObject experimentJson) {
         Gson gson = new Gson();
         JsonParser parser = new JsonParser();
 
-        String stringConditions = experimentJson.get("audienceConditions").getAsString();
+        JsonElement conditionsElement = experimentJson.get("audienceConditions");
 
-        JsonElement conditionsElement = parser.parse(stringConditions);
+        if (!conditionsElement.isJsonArray()) {
+            String stringConditions = experimentJson.get("audienceConditions").getAsString();
+
+            conditionsElement = parser.parse(stringConditions);
+        }
```

Comment:
I don't think we need this fallback at all for `experiment.audienceConditions` - I think it'll **always** be raw JSON. @aliabbasrizvi can you confirm?